package.json

```diff
@@ -147,6 +149,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.0",
     "canarist": "^1.2.1",
+    "commitizen": "^3.1.1",
```

Comment:
I would prefer to just install `cz-conventional-changelog` and have the config in the `package.json`, so that people who have `commitizen` installed globally may use it with our project.